### DISCUSSION TITLE
#100 is comming

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#193] [Ideas]
anything special from that occasion? 😃 
wold be nice!

**Comment by Beowolve** [comment]
CSSBattle Logo seams natural?

**Reply by chinchang** [reply]
😒🤨

**Reply by Beowolve** [reply]
I mean as target image, we could recreate the logo.
Also some Christmas images come to mind now.

**Reply by chinchang** [reply]
you had guessed it right! 
![image](https://user-images.githubusercontent.com/379918/147391928-4b5f9ccf-7669-414c-827b-eb8a3d7bdc1b.png)
